### DISCUSSION TITLE
freeU with SD1.5 on a TitanX 12Gb card

### DISCUSSION THREAD

**Discussion by lulu1315** [huggingface/diffusers#7753] [General]
hello. i'm trying to use freeU with SD1.5 on a TitanX 12Gb card and i get this message :
pipe.enable_freeu(s1=0.9, s2=0.2, b1=1.2, b2=1.4)
RuntimeError: cuFFT doesn't support signals of half type with compute capability less than SM_53, but the device containing input half tensor only has SM_52

the strange thing is that it runs perfectly when used with SDXL so i was wondering ..
(i'm using diffusers)

diffusers version: 0.27.0
Platform: Linux-5.15.0-94-generic-x86_64-with-glibc2.29
Python version: 3.8.0
PyTorch version (GPU?): 2.2.1+cu121 (True)
Huggingface_hub version: 0.21.4
Transformers version: 4.38.2
Accelerate version: 0.27.2
xFormers version: not installed

**Comment by tolgacangoz** [comment]
Hello, @lulu1315. When you were working with SD-XL, was its type `torch.float32` (this is the default)?

**Comment by lulu1315** [comment]
i just rechecked and sd15 and sdxl makes the same error when using freeU on an SM_52 when using fp16. sorry for the noise.
